File: 0485-max-consecutive-ones/0485-max-consecutive-ones.java
class Solution {
    public int findMaxConsecutiveOnes(int[] nums) {
        
        int cnt = 0;
        int maxi = Integer.MIN_VALUE;
        for(int ele : nums){
            if(ele != 0) cnt++;
            else cnt=0;
            maxi = Math.max(maxi,cnt);
        }
        return maxi;
    }
}

File: 0485-max-consecutive-ones/0485-max-consecutive-ones.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c=ones=0
        for i in range(len(nums)):
            if(nums[i]==1):
                c+=1
            elif(nums[i]==0):
                c=0
            ones = max(c,ones)
        return ones
        